Dockerfile

```diff
@@ -7,9 +7,9 @@ COPY entrypoint.sh /usr/local/bin
 
 WORKDIR /pack
 
-RUN wget https://github.com/buildpack/pack/releases/download/v0.3.0/pack-v0.3.0-linux.tgz
+RUN wget https://github.com/buildpacks/pack/releases/download/v0.6.0/pack-v0.6.0-linux.tgz
```

Comment:
Is there any way to make the version a variable?

Looks like it's already up to `v0.8.1` as of today: https://buildpacks.io/docs/install-pack/#linux

Reply:
This is a good point. I'll open a separate PR to look at the options for keeping up to date.